stellarphot/core.py

```diff
@@ -1295,9 +1297,10 @@ def passband_columns(self, passbands=None, transformer=None):
         # Add the metadata back to the table
         return_table.meta.update(metadata)
 
+        transformer_kwargs = transformer_kwargs or {}
```

Comment:
Clever, had to stare at this a bit to figure it out.  I might use it in my class when introducing booleans.